charts/firefly/templates/deployment.yaml

```diff
@@ -39,10 +39,11 @@ spec:
         ports:
         - containerPort: 8080
         volumeMounts:
-        - mountPath: /firefly/workarea
-          name: firefly-workarea
+        - mountPath: /firefly/shared-workarea
```

Comment:
To make sure that I understand properly:

If a filesystem / mount point is NOT mentioned here, e.g., `/firefly/workarea` after this change, it will get created as a container-local filesystem?

Reply:
Correct